### PR TITLE
minor refactoring and code fixes

### PR DESCRIPTION
**Pull request by A91y**

### CODE REVIEW

**Comment by A91y** [comment]
@Mr-Sunglasses Please review the PR.

File: src/enveasy/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from enveasy.cli import app
```
Comment (Mr-Sunglasses) [inline src/enveasy/__init__.py:1]: No Need , we can remove it

File: src/enveasy/parser.py [hunk 2]
```diff
@@ -1,19 +1,21 @@
 import toml
 
 
-def parse_envs_from_toml(file_path):
+def parse_envs_from_toml(file_path, debug=False):
```
Comment (Mr-Sunglasses) [inline src/enveasy/parser.py]: Why we are using debug ??

**Review by Mr-Sunglasses — CHANGES_REQUESTED** [review]
Add docs, then we can merge it

File: src/enveasy/parser.py (hunk 2 above)
Comment (A91y) [reply src/enveasy/parser.py]: > Why we are using debug ??

it was for initial testing, removing it...

File: src/enveasy/__init__.py (hunk 1 above)
Comment (A91y) [reply src/enveasy/__init__.py:1]: > No Need , we can remove it

@Mr-Sunglasses if this is removed, how will enveasy will reference to it, when used via terminal?

**Comment by A91y** [comment]
> Add docs, then we can merge it

Done!

**Review by Mr-Sunglasses — APPROVED** [review]

**Comment by Mr-Sunglasses** [comment]
Thanks 👍

**Pull request merged**